packages/lib/src/vis/shared/TooltipMesh.tsx

```diff
@@ -22,7 +23,8 @@ function TooltipMesh(props: Props) {
   const { width, height } = useThree((state) => state.size);
 
   // Scales to compute data coordinates from unprojected mesh coordinates
-  const { worldToData, worldToHtml } = useAxisSystemContext();
+  const { worldToData } = useAxisSystemContext();
+  const worldToHtml = useWorldToHtml();
```

Comment:
Damn, this isn't great. We're now calling `useFrameRendering` in the tooltip... I hadn't thought of the cases where `worldToHtml` is invoked from inside a callback like an event handler. :disappointed: